Server/src/app.ts

```diff
@@ -59,6 +62,24 @@ app.use('/token', cors(), issueToken);
  */
 app.use('/userConfig', cors(), userConfig);
 
+/**
+ * route: /createRoom
+ * purpose: Calling: create a new room
+ */
+app.use('/createRoom', cors(), createRoom);
+
+/**
+ * route: /addUserToRoom
+ * purpose: Calling: add user to room with the given role
+ */
+app.use('/addUserToRoom', cors(), addUserToRoom);
+
+/**
+ * route: /getLogUploadData
+ * purpose: Get tokens and endpoints for uploading logs to Azure Blob Storage
+ */
+app.use('/uploadToAzureBlobStorage', cors(), uploadToAzureBlobStorage);
+
```

Comment:
Are these needed in hero chat for stable? (same with associated file/helper functions)